modules/oc-mirror-workflows-delete-v2.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-workflows-delete-v2_{context}"]
+= Deleting the images from disconnected environment
+
+In v2, a new input type called `DeleteImageSetConfiguration` has been introduced. This is used to configure image deletion, preventing the accidental deletion of required or deployed images by avoiding the use of an `ImageSetConfig`.
+
+The new sub command allows the users to execute the delete workflow with the `DeleteImageSetConfiguration`.
+
+[NOTE]
+====
+In v2, automatic pruning is no longer performed. In v1, when an item (such as an operator) was removed from the image set configuration, it would be automatically pruned from the container registry. To prevent unintended deletions, v2 introduces a specific file named DeleteImageSetConfiguration. 
+====
+
+Using the DeleteImageSetConfiguration below as an example, oc-mirror will remove: 
+* All images of {product-title} release 4.13.3 .
```

Comment:
@subhtk - bullet points are not showing correctly

Reply:
Let me check again..